backend/tests/unit/notImplemented.ts

```diff
@@ -0,0 +1,3 @@
+export function notImplemented(): never {
+  throw new Error('Not implemented');
```

Comment:
Maybe add a dedicated type `NotImplementedError`, so we can branch if we need to implement a special case anywhere to handle this.

Reply:
It's just there for mocking reasons atm (use this mock if you don't wan't to implement a method).
It Is not used in the real app, so i don't think we need such a type here